src/components/layout/sidebar/Sidebar.vue

```diff
@@ -62,17 +59,10 @@
         }
       }
     },
-
-    watch: {
```

Comment:
Hm, we want menu to stay expanded after page refresh. Please refer to `http://vuestic.epicmax.co/#/ui/buttons` as an example.